website/docs/docs/build/packages.md

```diff
@@ -157,9 +157,60 @@ packages:
 
 Where `name: 'dbt_utils'` specifies the subfolder of `dbt_packages` that's created for the package source code to be installed within.
 
-### Private packages
+## Private packages
 
-#### SSH Key Method (Command Line only)
+### Native private packages <Lifecycle status='beta'/> 
+
+dbt Cloud supports private packages from [supported](#prerequisites) Git repos leveraging an exisiting [configuration](/docs/cloud/git/git-configuration-in-dbt-cloud) in your environment. Previously, you had to configure a [token](#git-token-method) to retrieve packages from your private repos.  
+
+#### Prerequisites
+
+Native support for private packages is supported on the following services:
```

Comment:
for this whole 'prerequisites' section, might be helpful to organize it all in bullets and link to the account integrations page. like this:

```
#### Prerequisites

- The following providers natively support private packages:
  - [GitHub](https://docs-getdbt-com-git-private-packages-dbt-labs.vercel.app/docs/cloud/git/connect-github)
  - [Azure DevOps](https://docs-getdbt-com-git-private-packages-dbt-labs.vercel.app/docs/cloud/git/connect-azure-devops)
  - GitLab support is coming soon.

- To use this feature, you must have a supported Git provider configured in the [Integrations section](/docs/cloud/account-integrations) of your Account settings.
```